charts/gardener-extension-shoot-lakom-service/templates/configmap-extension-config.yaml

```
@@ -19,6 +19,7 @@ data:
     useOnlyImagePullSecrets: {{ .Values.controllers.useOnlyImagePullSecrets }}
     allowUntrustedImages: {{ .Values.controllers.allowUntrustedImages }}
     allowInsecureRegistries: {{ .Values.controllers.allowInsecureRegistries }}
+    deploySeedResources: {{ get .Values.gardener.seed.annotations "service.lakom.extensions.gardener.cloud/deploySeedResources" }}
```

The example needs to be aligned with the changes in the API.

```suggestion
      enabled: true
    useOnlyImagePullSecrets: {{ .Values.controllers.useOnlyImagePullSecrets }}
    allowUntrustedImages: {{ .Values.controllers.allowUntrustedImages }}
    allowInsecureRegistries: {{ .Values.controllers.allowInsecureRegistries }}
```